packages/table/src/TableHeader.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  variant as getVariantStyles,
+  VariantProps,
+  sx,
+  Theme,
+  css,
+  SerializedStyles,
+  styled,
+} from '@kodiak-ui/core'
+import { tableRoot } from './Table'
+
+export type TableHeaderProps = React.HTMLAttributes<
+  HTMLTableHeaderCellElement
+> &
+  VariantProps
+
+export function base({ theme }: { theme: Theme }): SerializedStyles {
+  return css({
```

Comment:
Do we want to get the heading to have the default background gray.1, the pl: 4 (16px) and vertical borders here so that the reference style is close to the table designs?

Reply:
@alexanderchan not right here. We can update that in the theme though. I don't want to default to the Jilt styles when possible due to the fact that this should be used anywhere.